docs/usage/cli.md

```diff
@@ -20,6 +20,14 @@ available from the command line.
 $ npm install --global babel-cli
 ```
 
+Out of the box, Babel (since v6) doesn't perform any transformation, it only validates the syntax. You may want to install [plugins](/docs/plugins) or more likely whole [presets](/docs/plugins/#presets) before going further. For the sake of the example, we'll use the [es2015](/docs/plugins/preset-es2015/) preset.
+
+```sh
+$ npm install babel-preset-es2015
```

Comment:
For the presets, locally, but not saved, yes, that's what I'm proposing.

The file whose diff we're discussing instructs to install babel-cli globally. This is actually visible in the snippet [above](https://github.com/babel/babel.github.io/pull/610/files#diff-2f91ae7edae087838435be4adf7bc557L20).